cibuild.sh

```diff
@@ -24,6 +24,11 @@ apps=$WD/../apps
 tools=$WD/../tools
 prebuilt=$WD/../prebuilt
 
+# Python User Env
+PIP_USER=yes
+PYTHONUSERBASE=$WD/../pylocal
```

Comment:
How about change the path to:
PYTHONUSERBASE=$prebuilt/pylocal
like other tools

Reply:
done.